Analyzer/include/AnalyzeLepTrigger.h

```diff
@@ -28,6 +29,9 @@ class AnalyzeLepTrigger
     void printTriggerList( const std::vector<std::string>& TriggerNames );
     void doesTriggerExist( std::vector<std::string>& myTrigTestVector, const std::vector<std::string>& TriggerNames, const std::vector<int>& TriggerPass );
```

Comment:
I think this function can be moved to baseline or removed

Reply:
Removed.